README.md

```diff
@@ -169,6 +169,7 @@ The following table lists all valid connection properties:
 | CLIENT_CONFIG_FILE             | No       | The location of the client configuration json file. In this file you can configure easy logging feature.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
 | ALLOWUNDERSCORESINHOST         | No       | Specifies whether to allow underscores in account names. This impacts PrivateLink customers whose account names contain underscores. In this situation, you must override the default value by setting allowUnderscoresInHost to true.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
 | QUERY_TAG                      | No       | Optional string that can be used to tag queries and other SQL statements executed within a connection. The tags are displayed in the output of the QUERY_HISTORY , QUERY_HISTORY_BY_* functions.<br/> To set QUERY_TAG on the statement level you can use SnowflakeDbCommand.QueryTag.                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| ALLOW_SSO_TOKEN_CACHING        | No       | Specifies whether to cache tokens and use them for SSO authentication.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
you need to update the PR with changes from master - this table was moved to another file.

Reply:
I see that it's in `Connecting.md` now so I put this line there